src/components/Enumerate.tsx

```diff
@@ -44,9 +47,14 @@ class Enumerate extends React.Component<Props> {
   }
 }
 
-export default connect<StateProps, DispatchProps, {}, AppState>(
-  state => ({
+export const mapStateToProps = (state: AppState) => {
+  return {
     devices: state.device.devices,
-  }),
+    activeWallet: state.wallet.activeWallet
+  }
```

Comment:
My beef with this way of doing it is that the relationship with `StateProps` is implicit here. You could add a bunch unused props in here that aren't in StateProps, and TS won't yell at you for that. This could be fixed by adding `: StateProps` to the return type of this function, but then the benefit of doing this vs the previous way is just stylistic.

Reply:
I can add unused props into the original implementation and my TS isn't yelling at me. Have I misunderstood? 

```
export default connect<StateProps, DispatchProps, {}, AppState>(
  state => ({
    devices: state.device.devices,
    flibbity: "bibbity"
  }),
  { startDeviceScan, stopDeviceScan },
)(Enumerate);

```